### ISSUE TITLE
cargo scripts

### ISSUE THREAD

**Issue by vschwaberow** [mailtutan/mailtutan#56]
Isn't it better to replace the Makefile by using Cargo scripts?

https://doc.rust-lang.org/cargo/reference/build-scripts.html

**Comment by stappersg** [comment]
On Sat, Jun 17, 2023 at 11:03:36PM -0700, Volker Schwaberow wrote:
> Isn't it better to replace the Makefile by using Cargo scripts?

That confusion is legit.

**Comment by stappersg** [comment]
On Sun, Jun 18, 2023 at 12:22:01AM -0700, Geert Stappers wrote:
> On Sat, Jun 17, 2023 at 11:03:36PM -0700, Volker Schwaberow wrote:
> > Isn't it better to replace the Makefile by using Cargo scripts?
> 
> That confusion is legit.
> 

Merge requests #59 and #57 will take away that confusion.


Regards
Geert Stappers
-- 
Silence is hard to parse

**Comment by mohsen-alizadeh** [comment]
The makefile is just a collection of dev utilities and I used to do some different stuff like building docker image, pushing the image, building and ...
